### DISCUSSION TITLE
Clear conditional element values when hidden

### DISCUSSION THREAD

**Discussion by deanCompRSA** [vueform/vueform#341] [Questions]
For elements with conditions, how can I clear their value when they get hidden?

**Comment by adamberecz** [comment]
You can use the `clear()` method of either [Vueform](https://vueform.com/reference/vueform#method-clear) or the [element](https://vueform.com/reference/text-element#method-clear). Here's an example:
https://stackblitz.com/edit/github-fqkqfc?file=src%2FApp.vue
